features/apps/src/main/feature/feature.xml

```diff
@@ -37,15 +37,13 @@
     <repository>mvn:org.codice.ddf.spatial/spatial-app/${project.version}/xml/features</repository>
     <repository>mvn:ddf.catalog.solr/solr-app/${project.version}/xml/features</repository>
     <repository>mvn:org.codice.opendj.embedded/opendj-embedded-app/${opendj-embedded.app.version}/xml/features</repository>
-    <repository>mvn:org.codice.ddf/ui/${project.version}/xml/features</repository>
 
     <feature name="ddf-boot-features" version="${project.version}"
              description="Features that will be started on container startup.">
         <!--Added prerequisite to kernel since it starts up features that must be started before others such as wrap and blueprint -->
         <feature prerequisite="true">kernel</feature>
         <feature>secure-boot</feature>
         <feature>ddf-branding</feature>
-        <feature>ui</feature>
```

Comment:
For UI extensibility, we need to be able to replace the ui feature at the distribution level. Having the UI here makes it impossible to remove because [downstream distributions](https://github.com/codice/alliance/blob/master/distribution/alliance/src/main/filtered-resources/etc/org.apache.karaf.features.cfg#L56) include this feature.